app/javascript/packages/components/troubleshooting-options.jsx

```diff
@@ -11,17 +11,20 @@ import { BlockLink } from '@18f/identity-components';
 /**
  * @typedef TroubleshootingOptionsProps
  *
- * @prop {string} heading
+ * @prop {string} headingTag
+ * @prop {string} headingText
  * @prop {TroubleshootingOption[]} options
  */
 
 /**
  * @param {TroubleshootingOptionsProps} props
  */
-function TroubleshootingOptions({ heading, options }) {
+function TroubleshootingOptions({ headingTag = 'h2', headingText, options }) {
+  const HeadingTag = headingTag;
+
   return (
```

Comment:
I think we can rename this so we don't need the separate assignment

```suggestion
function TroubleshootingOptions({ headingTag: HeadingTag = 'h2', headingText, options }) {
  return (
```

Reply:
It is not currently used but will be in the future. I can cut it out but it'll be needed when the rest of the IAL2 flow is moved into react.